datastore-v1-proto-client/src/test/java/com/google/datastore/v1/client/RemoteRpcTest.java

```diff
@@ -174,7 +174,7 @@ public void testHttpHeaders_expectE2eChecksumHeader() throws IOException {
         httpRequest
             .getHeaders()
             .getFirstHeaderStringValue(EndToEndChecksumHandler.HTTP_REQUEST_CHECKSUM_HEADER);
-    assertEquals(32, header.length());
+    assertEquals(8, header.length());
```

Comment:
This seems an odd change. Is the crc32c hash that much smaller?

Reply:
CRC32C is 8 bytes only.

I would have preferred to use MD5 but internal review with ESF team pointed out that CRC32C is preferred (partly because ESF already uses CRC32C for internal checksum computation and this change would help cut down on rec-computation of checksum)

Reply:
we have also recently been moving to crc32c for other libs, as the computation should be faster for streams.

Reply:
oh ok - then moving to CRC32C in this client is probably acceptable.